x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,116 @@ export const PolicyList = React.memo(() => {
   const columns: Array<EuiTableFieldDataColumnType<Immutable<PolicyData>>> = useMemo(
     () => [
       {
-        field: 'name',
+        field: '',
         name: i18n.translate('xpack.siem.endpoint.policyList.nameField', {
           defaultMessage: 'Policy Name',
         }),
         // eslint-disable-next-line react/display-name
-        render: (value: string, item: Immutable<PolicyData>) => {
+        render: (item: Immutable<PolicyData>) => {
           const routePath = getManagementUrl({
             name: 'policyDetails',
             policyId: item.id,
             excludePrefix: true,
           });
           const routeUrl = getManagementUrl({ name: 'policyDetails', policyId: item.id });
-          return <PolicyLink name={value} route={routePath} href={routeUrl} />;
+          return (
+            <>
```

Comment:
When I first did something like this in Ingest, I meant for the following behaviours to be true:

- Label is truncated (does not span more than 1 line)
- Revision is always shown (even if label is long/trunctated)

Do your changes achieve that as well?

Reply:
good call, made a few adjustments.  Will look like this after I push

![image](https://user-images.githubusercontent.com/56395104/83445801-a5be3600-a41b-11ea-8a3b-b5e9c43e650d.png)